.travis.yml

```
@@ -0,0 +1,9 @@
+services:
+- docker
+
+script:
+  - docker build --target test --tag my-test-image .
+  - docker run my-test-image todo_app/test_tasks.py
+  - docker run -e TRELLO_KEY=$TRELLO_KEY -e TRELLO_TOKEN=$TRELLO_TOKEN -e TRELLO_BOARD=$TRELLO_BOARD -e TRELLO_TODO_LIST_ID=$TRELLO_TODO_LIST_ID -e TRELLO_DOING_LIST_ID=$TRELLO_DOING_LIST_ID -e TRELLO_DONE_LIST_ID=$TRELLO_DONE_LIST_ID my-test-image todo_app/test_client.py
```

Your integration test shouldn't use your real environment variables. That way you can't accidentally end up talking to a real service if you forget to mock something.
```suggestion
  - docker run --env-file .env.test my-test-image todo_app/test_client.py
```